ui/app/serializers/cluster.js

```diff
@@ -1,6 +1,9 @@
 import RESTSerializer, { EmbeddedRecordsMixin } from '@ember-data/serializer/rest';
 import { assign } from '@ember/polyfills';
 import { decamelize } from '@ember/string';
+import IdentityManager from '../utils/identity-manager';
+
+const uuids = new IdentityManager();
```

Comment:
I was running into an issue where `cluster_id` was not defined and the replication attributes need a unique id or the store will throw an error so I adapted the mirage identity manager into a utility and use it here.